ONSdigital/sdx-collect
# Python async consumer

**Changes**
Convert sdx-collect to use an async consumer, similar to sdx-downstream
Calls new sdx-mock-receipt
Added unit tests

**How to test**
Start the app as part of compose setup. Check that the full workflow occurs.
Toggle the environment variable for skipping receipt.

**Who can test**
Anyone but @elliott-jenkins
